src/main/jbake/content/blog/2016/04/21/datamining-first-release.adoc

```diff
@@ -0,0 +1,84 @@
+= Hawkular Data Mining 0.1.0.Final Released
+Pavol Loffay
+2016-04-21
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, hawkular, datamining
+
+I am happy to announce first release 0.1.0.Final of the Hawkular Data Mining project. It contains several time series
+models and utility classes for time series modelling.
+
+=== Time series models
+  - Simple exponential smoothing
+  - Double exponential smoothing (Holt's linear trend)
+  - Seasonal triple exponential smoothing (Holt Winters)
+  - Simple moving average (Weighted moving average)
+  - AutomaticForecaster - which automatically selects the best model
+  - Parameters of all models are estimated using maximum likelihood estimation and models are designed for online
+  learning
+
+=== Time series manipulation & Statistics
+  - Augmented Dickey-Fuller test
+  - Autocorrelation function (ACF)
+  - Time series decomposition
+  - Time series lagging
+  - Time series differencing
+  - Automatic period identification
+
+== Integration into Hawkular
+
+The integration into Hawkular can be found in _datamining_ branch in the main Hawkular repository.
+Predictive charts are located in Explorer tab. There can be enable predictions for any number of steps in the future for
+ any metric being collected. Prediction engine automatically selects the best model for given time series. Currently it
+selects from simple, double and triple exponential smoothing models. In the following charts are
```

Comment:
In the following charts are showed predictions produced by these models.
In the following charts predictions produced by these models are showed .